experimenter/tests/integration/nimbus/conftest.py

```diff
@@ -37,15 +38,6 @@
     BaseExperimentApplications.FIREFOX_FENIX.value: (
         "#/buckets/main-workspace/collections/nimbus-mobile-experiments/simple-review"
     ),
```

Comment:
Wait why remove iOS?

Reply:
CPU usage with 4 parallel runs: 
<img width="1394" height="418" alt="image" src="https://github.com/user-attachments/assets/168ed5a4-a495-4c49-9cfe-266c36235eeb" />

Reply:
No I know, that's why we added it in the first place becuase there was a period where we only tested desktop in the integration flows, and there WAS a configuration error specifically for iOS, and we didn't notice until someone tried to launch it and discovered the RS publishing was busted, so I think we should have at least ONE test that walks through the entire flow for every application.  Doesn't have to be all of them, but at least one.